prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_SR_2_20231024-0002.hl7

```diff
@@ -1 +1,12 @@
OBR|1|5f0d29c2-4181-4a1e-8774-b88338602681^Carl Potts^12D3456789^CLIA|5f0d29c2-4181-4a1e-8774-b88338602681^Carl Potts^12D3456789^CLIA|95941-1^^LN|||20231018220520+0000|||||||||^Smith^Georgia|^WPN^PH^^1^248^5551234^^^^^(248) 555 1234|||||20231018222022+0000|||F
OBX|1|CWE|85477-8^^LN^^^^^^Flu A||260373001^Detected^SCT|||A^Abnormal^HL70078^^^^2.7|||F|||20231018220520+0000|12D3456789^Carl Potts^CLIA||^Alinity m|00884999048034|20231018222020+0000||||Carl Potts^L^^^^&2.16.840.1.113883.4.7&ISO^^^^12D3456789|400 Main Street^Suite 100^Birmingham^AL^35228^USA
OBX|2|CWE|85478-6^^LN^^^^^^Flu B||260373001^Detected^SCT|||A^Abnormal^HL70078^^^^2.7|||F|||20231018220520+0000|12D3456789^Carl Potts^CLIA||^Alinity m|00884999048034|20231018222020+0000||||Carl Potts^L^^^^&2.16.840.1.113883.4.7&ISO^^^^12D3456789|400 Main Street^Suite 100^Birmingham^AL^35228^USA
OBX|3|CWE|94500-6^^LN^^^^^^COVID-19||260373001^Detected^SCT|||A^Abnormal^HL70078^^^^2.7|||F|||20231018220520+0000|12D3456789^Carl Potts^CLIA||^Alinity m|00884999048034|20231018222020+0000||||Carl Potts^L^^^^&2.16.840.1.113883.4.7&ISO^^^^12D3456789|400 Main Street^Suite 100^Birmingham^AL^35228^USA
OBX|4|CWE|95419-8^Has symptoms related to condition of interest^LN^^^^2.69^^Has symptoms related to condition of interest||UNK^unknown||||||F||||12D3456789^Carl Potts^CLIA||||||||||||||QST
OBX|5|CWE|95418-0^Employed in a healthcare setting^LN^^^^2.69^^Employed in a healthcare setting||N^No^HL70136||||||F||||12D3456789^Carl Potts^CLIA||||||||||||||QST
OBX|6|CWE|95421-4^Resides in a congregate care setting^LN^^^^2.69^^Resides in a congregate care setting||N^No^HL70136||||||F||||12D3456789^Carl Potts^CLIA||||||||||||||QST
SPM|1|5f0d29c2-4181-4a1e-8774-b88338602681&Carl Potts&12D3456789&CLIA^5f0d29c2-4181-4a1e-8774-b88338602681&Carl Potts&12D3456789&CLIA||697989009^Anterior nares swab^SCT^^^^2.67^^Anterior nares swab|||||||||||||20231018220520+0000|20231018220520+0000
\ No newline at end of file
+MSH|^~\&|CDC PRIME - Atlanta^2.16.840.1.114222.4.1.237821^ISO|CDC PRIME REPORTSTREAM^10DRPTSTRM^CLIA|ECLRS|NYSDOH|20231018222022+0000||ORU^R01^ORU_R01|5f0d29c2-4181-4a1e-8774-b88338602681|T|2.5.1|||||USA|UNICODE UTF-8|||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
+SFT|Centers for Disease Control and Prevention|0.2-SNAPSHOT|PRIME ReportStream|0.2-SNAPSHOT||20231018210603+0000
+PID|1||b023143c-b30d-4567-b651-be966ebd5532^^^Carl Potts&12D3456789&CLIA^PI^Carl Potts&12D3456789&CLIA||West^Theodora^Stacy^^^^L||19660304|F||ASKU^refused^^^^^2.5.1^^refused|40405 Lionel Lakes^^New Savannah^NY^10001^USA||^PRN^PH^^1^410^7542890^^^^^(410) 754 2890~^NET^Internet^Theodora4@yahoo.com|||||||||N^Not Hispanic or Latino^HL70189^^^^2.9^^Not Hispanic or Latino
```

Comment:
The only real change here was `WPN`->`NET` in PID.13 based on @victor-chaparro 's advice.